DigitalOffice/Tests/UserServiceTests/RepositoriesTests/UserRepositoryTests.cs

```diff
@@ -9,7 +9,7 @@
 using LT.DigitalOffice.UserService.Repositories;
 using LT.DigitalOffice.UserService.Repositories.Interfaces;
 
-namespace LT.DigitalOffice.UserServiceUnitTests.Repositories
+namespace LT.DigitalOffice.UserServiceUnitTests.RepositoriesTests
```

Comment:
It is necessary to change the name from RepositoriesTests to Repositories.

Reply:
Fixed